tests/proxy_view_tests.cpp

```diff
@@ -10,8 +10,8 @@ namespace proxy_view_tests_details {
 struct TestFacade : pro::facade_builder
     ::add_convention<pro::operator_dispatch<"+=">, void(int)>
     ::add_convention<utils::spec::FreeToString, std::string(), std::string() const>
-    ::add_view<TestFacade>
-    ::add_view<const TestFacade>
+    ::support_view
```

Comment:
Q: How about making `support_const_view` be implied by `support_view`?

Reply:
I tend not to, since providing conversion to `proxy_view<const F>` makes no sense if there is no `const` convention at all, while it increases binary size. The consideration is similar to the separation of `support_format` and `support_wformat`.

Reply:
that makes sense.